test/stat_tracker_test.rb

```diff
@@ -12,15 +12,27 @@
 class StatTrackerTest < MiniTest::Test
 
   def setup
-    @stat_tracker = StatTracker.new
+    game_path = './data/game.csv'
+    team_path = './data/team_info.csv'
+    game_teams_path = './data/game_teams_stats.csv'
+
+    locations = {
+      games: game_path,
+      teams: team_path,
+      game_teams: game_teams_path
+    }
+
+    @stat_tracker = StatTracker.from_csv(locations)
   end
 
-  def test_it_exists
+  def test_stat_tracker_exists
     assert_instance_of StatTracker, @stat_tracker
   end
 
   def test_add_teams_and_count
-    assert_equal 33, StatTracker.from_csv
+    assert_equal 33, @stat_tracker.teams.count
+    assert_equal 7441, @stat_tracker.games.count
+    assert_equal 14882, @stat_tracker.game_team_stats.count
   end
```

Comment:
We can add further tests to confirm that the StatTracker initialize is creating an instance for each row of data for these CSV files. 

example: 
stat_tracker.games.each do |game|
  assert_instance_of Game, game
end